embedded-workshop-book/src/beginner-next-steps.md

```diff
@@ -15,9 +15,10 @@ The `Radio` abstraction supports 2 CCA modes: `CarrierSense` and `EnergyDetectio
 Here are some things for you to try out:
 - First, read the section 6.20.12.4 of the nRF52840 Product Specification, which covers the nRF52840's implementation of CCA.
 
-- Disconnect the Dongle. Write a program for the DK that scans and reports the energy levels of all valid 802.15.4 channels. In your location which channels have high energy levels when there's no ongoing 802.15.4 traffic? If you can, use an application like [WiFi Analyzer] to see which WiFi channels are in use in your location. Compare the output of WiFiAnalyzer to the values you got from  `energy_detection_scan`. Is there a correspondence? (Note that WiFi channels don't match in frequency with 802.15.4 channels; some mapping is required to convert between them)
+- Disconnect the Dongle. Write a program for the DK that scans and reports the energy levels of all valid 802.15.4 channels. In your location which channels have high energy levels when there's no ongoing 802.15.4 traffic? If you can, use an application like [WiFi Analyzer] to see which WiFi channels are in use in your location. Compare the output of WiFiAnalyzer to the values you got from  `energy_detection_scan`. Is there a correspondence? Note that WiFi channels don't match in frequency with 802.15.4 channels; some mapping is required to convert between them -- check [this resource] for more details about co-existence between 802.15.4 and WiFi.
 
 [WiFi Analyzer]: https://play.google.com/store/apps/details?id=com.farproc.wifi.analyzer&hl=en
+[this resource]: https://inet.omnetpp.org/docs/showcases/wireless/coexistence/doc/
```

Comment:
accessibility nitpick: consider writing something like "the illustration in this article" so that the link is easy to find using a screen reader (and people know that there's an illustration behind it as a side effect)
see https://www.washington.edu/accessibility/links/